<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>732</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>775</width>
    <height>732</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(43, 43, 43);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>781</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(236, 236, 234);</string>
    </property>
    <property name="text">
     <string>Configuración del Sistema de Riego</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="listaPuertos">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(236, 236, 234);</string>
    </property>
    <item>
     <property name="text">
      <string>Lista de Puertos</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="openPort">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(236, 236, 234);</string>
    </property>
    <property name="text">
     <string>Abrir Puerto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closePort">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(236, 236, 234);</string>
    </property>
    <property name="text">
     <string>Cerrar Puerto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scanPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(236, 236, 234);</string>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>167</y>
      <width>751</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="Line" name="line_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(233, 229, 129);</string>
         </property>
         <property name="text">
          <string>Configuración Alarma de Riego</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTimeEdit" name="tiempoConfig">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(236, 236, 234);</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="displayFormat">
          <string>h:mm AP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendAlarm">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(236, 236, 234);</string>
         </property>
         <property name="text">
          <string>Enviar Hora</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="readAlarm">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(236, 236, 234);</string>
         </property>
         <property name="text">
          <string>Leer Datos Alarma</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="4">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(233, 229, 129);</string>
         </property>
         <property name="text">
          <string>Lectura Variables Ambientales</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="readTemperature">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(236, 236, 234);</string>
         </property>
         <property name="text">
          <string>Leer Temperatura</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="readHumidity">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(236, 236, 234);</string>
         </property>
         <property name="text">
          <string>Leer Humedad </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="controlValve">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(236, 236, 234);</string>
         </property>
         <property name="text">
          <string>Activar Valvula</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(233, 229, 129);</string>
         </property>
         <property name="text">
          <string>Configuración RTC</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeRTC">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(236, 236, 234);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxDias">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(236, 236, 234)</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="maxVisibleItems">
          <number>7</number>
         </property>
         <item>
          <property name="text">
           <string>Lunes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Martes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Miercoles</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Jueves</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Viernes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sabado</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Domingo</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendRTC">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(236, 236, 234);</string>
         </property>
         <property name="text">
          <string>Enviar Fecha y Hora</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="readRTC">
         <property name="font">
          <font>
           <family>Open Sans</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(236, 236, 234);</string>
         </property>
         <property name="text">
          <string>Leer Datos RTC</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="3">
      <widget class="Line" name="line_3">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>751</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(236, 236, 234);</string>
    </property>
    <property name="text">
     <string>Información Recibida</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="rxResult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>464</y>
      <width>751</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(233, 229, 129);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
